.nuke/build.schema.json

```diff
@@ -161,10 +161,8 @@
           "items": {
             "type": "string",
             "enum": [
-              "Docker_Win_NET48",
-              "Docker_Win_NET481",
               "Docker_Win_NET8_0",
-              "EnforcedLocalMachine(Docker_Win_NET6_0)",
+              "EnforcedLocalMachine(Docker_Win_NET8_0)",
```

Comment:
Here and in Build.cs we still had net-6 mentioned. I fixed that one.
@a-ctor 
cc @Frazerus